src/net/sourceforge/kolmafia/session/ChoiceAdventures.java

```diff
@@ -6052,6 +6110,52 @@ public static final Option findOption(final Option[] options, final int decision
             .values()
             .toArray(new ChoiceAdventure[choiceToChoiceAdventure.size()]);
     Arrays.sort(CHOICE_ADVS);
+
+    // Log errors detected during class initialization. These are programming
+    // errors and the developer who inserted the bug should have seen the
+    // report on stdout, but, just in case...
+    if (duplicateChoiceAdventures.size() > 0) {
+      RequestLogger.printLine(
+          "Duplicate ChoiceAdventures: ("
+              + duplicateChoiceAdventures.stream()
+                  .map(String::valueOf)
+                  .collect(Collectors.joining(","))
+              + ")");
+    }
+    if (missingChoiceAdventureOptions.size() > 0) {
+      RequestLogger.printLine(
+          "Missing ChoiceAdventure Options: ("
+              + missingChoiceAdventureOptions.stream()
+                  .map(String::valueOf)
+                  .collect(Collectors.joining(","))
+              + ")");
+    }
+    if (missingChoiceAdventureDefaultProperties.size() > 0) {
+      RequestLogger.printLine(
+          "Missing ChoiceAdventure default peoperties: ("
+              + missingChoiceAdventureDefaultProperties.stream()
+                  .map(String::valueOf)
+                  .collect(Collectors.joining(","))
+              + ")");
+    }
+    if (duplicateChoiceSpoilers.size() > 0) {
+      RequestLogger.printLine(
+          "Duplicate ChoiceSpoilers: ("
+              + duplicateChoiceSpoilers.stream()
+                  .map(String::valueOf)
+                  .collect(Collectors.joining(","))
+              + ")");
+    }
+    if (duplicateChoiceCosts.size() > 0) {
+      RequestLogger.printLine(
+          "Duplicate ChoiceCosts: ("
+              + duplicateChoiceCosts.stream().map(String::valueOf).collect(Collectors.joining(","))
+              + ")");
+    }
+
+    // System.out.println("ChoiceSpoilers with defaults = " + choiceSpoilersWithDefaults.size());
```

Comment:
rather than having some commented out code, could we have e.g. a `test choices` command that prints them out when run in the CLI?

Reply:
I will do that. And, per my new top-level comment, I will probably not write a test for it - although I could, if you insist. ;)

Reply:
```
> test choicedefaults

There are 54 ChoiceSpoilers with defaults.
{2,3,6,8,9,10,11,12,26,27,28,29,45,86,87,88,89,123,125,127,129,152,162,276,277,278,279,280,281,282,283,284,285,286,287,288,289,290,503,504,505,506,507,549,550,551,552,553,554,583,873,888,889,1392}
There are 12 ChoiceSpoilers without defaults.
{235,240,250,251,252,255,259,264,267,268,914,1063}
```